src/sokoban/Game.java

```diff
@@ -6,54 +6,124 @@ public class Game extends Canvas {
     //
     // Game objects
     //
-    public class FlagLV implements Interfaces.IBlock {
+    public class Ground implements Interfaces.IBlock {
         @Override
         public void Draw(Graphics ctx, int x, int y) {
-            Color lv_red = new Color(140, 0, 0);
-            ctx.setColor(lv_red);
-            ctx.fillRect(x, y, Config.BLOCK_WIDTH, Config.BLOCK_HEIGHT / 3);
+           Color ground_brown = new Color(73, 50, 4);
+           Color nugget_yellow = new Color(118,87,25);
+           Color grass_green= new Color (31,82,15);
+            ctx.setColor(ground_brown);
+            ctx.fillRect(x, y, Config.BLOCK_WIDTH, Config.BLOCK_HEIGHT );
+            
+            ctx.setColor(nugget_yellow);
+            ctx.fillRect(x, y, 12, 9);
+            ctx.fillRect(x+32, y+12, 12, 9);
+            ctx.fillRect(x+48, y+48, 12, 9);
+            ctx.fillRect(x+3, y+24, 12, 9);
+            ctx.fillRect(x+22, y+40, 12, 9);
+            
+            ctx.setColor(grass_green);
+            ctx.fillRect(x+48, y+30, 12, 9);
 
-            ctx.setColor(Color.WHITE);
-            ctx.fillRect(x, y + Config.BLOCK_HEIGHT / 3 * 1, Config.BLOCK_WIDTH, Config.BLOCK_HEIGHT / 3);
-
-            ctx.setColor(lv_red);
-            ctx.fillRect(x, y + Config.BLOCK_HEIGHT / 3 * 2, Config.BLOCK_WIDTH, Config.BLOCK_HEIGHT / 3);
+           
         }
     }
 
-    public class FlagUA implements Interfaces.IBlock {
+    public class Brick implements Interfaces.IBlock {
         @Override
         public void Draw(Graphics ctx, int x, int y) {
-            ctx.setColor(Color.BLUE);
-            ctx.fillRect(x, y, Config.BLOCK_WIDTH, Config.BLOCK_HEIGHT / 2);
-
-            ctx.setColor(Color.YELLOW);
-            ctx.fillRect(x, y + Config.BLOCK_HEIGHT / 2, Config.BLOCK_WIDTH, Config.BLOCK_HEIGHT / 2);
+        	Color gray = new Color (115,113,110);
+        	Color nugget_gray = new Color(88,87,83);
+            ctx.setColor(gray);
+            ctx.fillRect(x, y, Config.BLOCK_WIDTH, Config.BLOCK_HEIGHT );
+            ctx.setColor(nugget_gray);
+            ctx.fillRect(x, y, 12, 9);
+            ctx.fillRect(x+24, y, 12, 9);
+            ctx.fillRect(x+48, y, 12, 9);  
+            
+            ctx.fillRect(x+12, y+12, 12, 9);
+            ctx.fillRect(x+36, y+12, 12, 9);
+            
+            ctx.fillRect(x, y+24, 12, 9);
+            ctx.fillRect(x+24, y+24, 12, 9);
+            ctx.fillRect(x+48, y+24, 12, 9);
+            
+            ctx.fillRect(x+12, y+36, 12, 9);
+            ctx.fillRect(x+36, y+36, 12, 9);
+            
+            ctx.fillRect(x, y+48, 12, 9);
+            ctx.fillRect(x+24, y+48, 12, 9);
+            ctx.fillRect(x+48, y+48, 12, 9);
+           
         }
     }
 
     //
     // Game
     //
+//    Обьявляю тут Массив, 
```

Comment:
![](https://media.tenor.com/RUJokzj0C2gAAAAd/speak-english-pulp-fiction.gif)

```suggestion
    // Declare array here
```